testlib/src/test/java/io/spine/testing/TestsTest.java

```diff
@@ -181,6 +185,102 @@ void fieldIsNotSet() {
             assertThrows(AssertionError.class,
                          () -> assertMatchesMask(Timestamp.getDefaultInstance(), fieldMask));
         }
+
+        @Nested
+        @DisplayName("with repeated fields")
+        class RepeatedFields {
+
+            private final PatientId patientId = newPatient();
+
+            @DisplayName("match existing repeated primitive fields")
+            @Test
+            void matchRepeatedPrimitiveFields() {
+                Prescription prescription = prescribeFromCold();
+                FieldMask fieldMask = FieldMaskUtil.fromFieldNumbers(Prescription.class, 1, 2);
```

Comment:
Integer literals should not be used here. The message classes generate `static` constants for the field tags.